include/h2o/memory.h

```diff
@@ -167,8 +167,10 @@ extern void *(*volatile h2o_mem__set_secure)(void *, int, size_t);
 /**
  * prints an error message and aborts
  */
-#define h2o_fatal(msg) h2o__fatal(__FILE__ ":" H2O_TO_STR(__LINE__) ":" msg)
-H2O_NORETURN void h2o__fatal(const char *msg);
+H2O_NORETURN void h2o__fatal(const char *msg, ...);
```

Comment:
Please set the function attribute that we have for other printf style vararg functions.

Reply:
Yes, Fixed.